### PR TITLE
Shorten shortDescs, updated descs, other minor text edits

### PR DESCRIPTION
**Pull request by Lusamine**
Tested on Firefox Quantum + Windows 10.  The only desc that still cuts off is Compost, and it loses only the period.

### CODE REVIEW

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -3329,6 +3329,21 @@ let BattleMovedex = {
...
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "For 5 turns, the weather becomes Acid Rain.  Pokemon that are not Poison-type take damage every turn.  Special Defense of Poison-type pokemon is multiplied by 1.5.  Poison moves ignore Steel immunity and Poison-type Pokemon can be poisoned.",
+		shortDesc: "5 turns: +Poison SpD, corrosive damage.",
+		id: "acidrain",
+		name: "Acid Rain",
+		isNonstandard: "Custom",
+		pp: 5,
+		priority: 0,
```
Comment (tmagicturtle) [inline data/mods/ssb/moves.js]: flags: {},
weather: 'acidrain',
onTryMove() {
	this.attrLastMove('[still]');
},
onPrepareHit(target, source) {
	this.add('-anim', source, 'Rain Dance', source);
},
secondary: null,

Comment (tmagicturtle) [inline data/mods/ssb/moves.js]: flags: {},
weather: 'acidrain',
onTryMove() {
	this.attrLastMove('[still]');
},
onPrepareHit(target, source) {
	this.add('-anim', source, 'Rain Dance', source);
},
secondary: null,

File: data/mods/ssb/moves.js [hunk 2]
```diff
@@ -3329,6 +3329,21 @@ let BattleMovedex = {
...
+		target: "self",
```
Comment (tmagicturtle) [inline data/mods/ssb/moves.js]:
```suggestion
		priority: 0,
		flags: {},
                weather: 'acidrain',
                onTryMove() {
                  this.attrLastMove('[still]');
                },
                onPrepareHit(target, source) {
                this.add('-anim', source, 'Rain Dance', source);
                },
                secondary: null,
```

File: data/mods/ssb/moves.js (hunk 1 above)
Comment (tmagicturtle) [inline data/mods/ssb/moves.js]:
```suggestion
		flags: {},
                weather: 'acidrain',
                onTryMove() {
                  this.attrLastMove('[still]');
                },
                onPrepareHit(target, source) {
                  this.add('-anim', source, 'Rain Dance', source);
                },
                secondary: null,
```

File: data/mods/ssb/moves.js (hunk 2 above)
Comment (tmagicturtle) [inline data/mods/ssb/moves.js]:
```suggestion
		flags: {},
                weather: 'acidrain',
                onTryMove() {
                  this.attrLastMove('[still]');
                },
                onPrepareHit(target, source) {
                  this.add('-anim', source, 'Rain Dance', source);
                },
                secondary: null,
```

File: data/mods/ssb/moves.js [hunk 3]
```diff
@@ -3329,6 +3329,22 @@ let BattleMovedex = {
...
+		basePower: 0,
+		category: "Status",
+		desc: "For 5 turns, the weather becomes Acid Rain.  Pokemon that are not Poison-type take damage every turn.  Special Defense of Poison-type pokemon is multiplied by 1.5.  Poison moves ignore Steel immunity and Poison-type Pokemon can be poisoned.",
+		shortDesc: "5 turns: +Poison SpD, corrosive damage.",
+		weather: 'acidrain',
+		id: "acidrain",
+		name: "Acid Rain",
+		isNonstandard: "Custom",
+		pp: 5,
+		priority: 0,
```
Comment (tmagicturtle) [inline data/mods/ssb/moves.js:3343]:
```suggestion
		priority: 0,
		flags: {},
                weather: 'acidrain',
                onTryMove() {
                  this.attrLastMove('[still]');
                },
                onPrepareHit(target, source) {
                  this.add('-anim', source, 'Rain Dance', source);
                },
                secondary: null,
```

File: data/mods/ssb/moves.js [hunk 4]
```diff
@@ -3329,6 +3329,30 @@ let BattleMovedex = {
...
+		flags: {},
+		weather: 'acidrain',
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Rain Dance', source);
+		},
+		secondary: null,
+		target: "self",
```
Comment (HoeenCoder) [inline data/mods/ssb/moves.js]:
```suggestion
		target: "all",
```

**Pull request merged**